src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,11 +1,13 @@
-2018-01-31 10:51:15 UTC
+
+Type dsc to start/install the DS Consolidated Console
+
+2018-03-09 17:32:10 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: kosta-bizetic
+GitHub user: Azure
```

Comment:
@sarvesh-s please pull from upstream and regenerate using generate.cmd
It has the fix to get you the commit id for your generation